scss/mixins/_type-mixins.scss

```diff
@@ -80,3 +80,7 @@
   font-weight: bold;
   padding: 0 0.3em;
 }
+
+@mixin t-upper {
```

Comment:
Oh, I mis-understood your question... feeling really silly now :D Yeah, using `@include` or `@extend` seems pretty silly for a single property, especially one like "upper" that would never really having any other styles besides `text-transform: uppercase`.

I'll remove it and use the property directly. I won't touch `t-hightlight`.